src/test/groovy/hudson/plugins/gradle/PropertyPassingIntegrationTest.groovy

```diff
@@ -1,5 +1,6 @@
 package hudson.plugins.gradle
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings
```

Comment:
Could you please not use Findbugs for Groovy integration tests? Add Codenarc instead!